test/MUXControlsTestApp/MUXControlsTestApp.Shared.projitems

```diff
@@ -152,6 +152,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <Content Include="$(MSBuildThisFileDirectory)Assets\Nuclear_symbol.svg" />
```

Comment:
>Nuclear_symbol [](start = 56, length = 14)

only using Nuclear_symbol, can you delete Victorian-Vortex?

Reply:
yes, thank you